src/whylogs/app/logger.py

```diff
@@ -159,45 +158,44 @@ def _intialize_profiles(self,
             {"full_profile": full_profile, "segmented_profiles": {}})
 
     def _set_rotation(self, with_rotation_time: str = None):
-        if with_rotation_time is not None:
-            self.with_rotation_time = with_rotation_time.lower()
-
-            m = re.match(r'^(\d*)([smhd])$', with_rotation_time.lower())
-            if m is None:
-                raise TypeError("Invalid rotation interval, expected integer followed by one of 's', 'm', 'h', or 'd'")
-
-            interval = 1 if m.group(1) == '' else int(m.group(1))
-            if m.group(2) == 's':
-                self.suffix = "%Y-%m-%d_%H-%M-%S"
-            elif m.group(2) == 'm':
-                interval *= 60  # one minute
-                self.suffix = "%Y-%m-%d_%H-%M"
-            elif m.group(2) == 'h':
-                interval *= 60 * 60  # one hour
-                self.suffix = "%Y-%m-%d_%H"
-            elif m.group(2) == 'd':
-                interval *= 60 * 60 * 24  # one day
-                self.suffix = "%Y-%m-%d"
-            else:
-                raise TypeError("Invalid rotation interval, expected integer followed by one of 's', 'm', 'h', or 'd'")
-            # time in seconds
-            current_time = int(datetime.datetime.utcnow().timestamp())
-            self.interval = interval * self.interval_multiplier
-            self.rotate_at = self.rotate_when(current_time)
+        if with_rotation_time is None:
```

Comment:
We're still going to need to merge my PR to update the documentation and get the unit test for the change to with_rotation_time

Reply:
we merged that one already